src/server.js

```diff
@@ -301,8 +326,13 @@ function setupRest(app) {
 						}
 						links.push(link);
 					}
-					res.setHeader('Link', links);
 				}
+				// Advertise support for SSE
+				let link = '<' + config.rest + '/sse/' + uuid + '>; ' +
+					'rel="urn:ietf:params:whep:ext:core:server-sent-events"; ' +
```

Comment:
Heads-up: I noticed there's some inconsistency about these rel values in the spec.

Opened murillo128/webrtc-http-egress-protocol#13 to discuss that and come up with a canonical answer, but this line and its parallel in web/watch.js may need to change.

Reply:
Thanks for the heads up! (and sorry for the late feedback, I was busy at the IETF meeting).